src/Microsoft.Extensions.Logging.Abstractions/ILoggerFactory.cs

```diff
@@ -19,9 +19,15 @@ public interface ILoggerFactory : IDisposable
         ILogger CreateLogger(string categoryName);
 
         /// <summary>
+        /// <para>
+        /// This method is being removed from the ILoggerFactory abstraction, to have similar behavior
```

Comment:
Recommend following a pattern closer to the guidelines: https://github.com/aspnet/Home/wiki/Engineering-guidelines#breaking-changes